snippets/react-snippets.code-snippets

```diff
@@ -114,6 +114,35 @@
         ],
         "description": "Add `role='radio'` to a checkable interactive control. Use radio in place of checkbox if only one item in a group can be checked. Add `aria-checked` to indicate the state of the checkbox."
     },
+    "TableAria": {
+        "prefix": [
+            "TableAria",
+        ],
+        "body": [
+            "aria-colcount=''",
+            "aria-rowcount={-1}",
+            "align={}",
+            "border={$2}",
+            "frame='{$1}'",
+            "width='{$2}'",
+            "bgcolor='{$1}'",
+        ],
+        "description": "Add `role='table'` to allow us represents tabular data information presented in a two-dimensional table comprised of rows and columns of cells containing data. "
+    },
+    "CellAria": {
+        "prefix": [
+            "CellAria",
+        ],
+        "body": [
+            "role='cell'",
```

Comment:
perfection!

Reply:
Sorry one thing i forgot to mention you shouldnt use bot the quotes and the brackets at the same time
quotes are for string values like aria-label for example.
and {} are for things like numbers

variable={$1} => variable={}
variableString='$1' => variable=''